cobalt/build/BUILD.gn

```diff
@@ -12,14 +12,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-action("cobalt_build_id") {
-  cobalt_version = exec_script("get_build_id.py", [], "trim string")
+action("cobalt_build_info") {
+  output = exec_script("get_build_info.py", [], "trim string")
+  build_info = string_split(output, "|")
```

Comment:
I don't think build info should be put in this action at all, I think it should be its own separate action with a script `write_build_info.py` that handles getting and writing all the appropriate values on its own, not passing them through GN

Reply:
To maintain backwards compatibility it won't touch anything related to build_id and will just sit parallel to it. There's some duplicated code, but that's almost a must because the git_rev is very closely tied to how we find build_id.

Doing it this way there's only a singular python script and no values need to pass through GN.